code/modules/fishing/fishing_spots.dm

```diff
@@ -47,8 +47,7 @@ ABSTRACT_TYPE(/datum/fishing_spot)
 	user.visible_message("[user] [pick("reels in", "catches", "pulls in", "fishes up")] a \
 	[pick("big", "wriggly", "fat", "slimy", "fishy", "large", "high-quality", "nasty", "chompy", "real", "wily")] \
 	[prob(80) ? "[fish.name]" : pick("one", "catch", "chomper", "wriggler", "sunovagun", "sucker")]!")
-	user.put_in_hand_or_drop(fish)
-	//fish.set_loc(get_turf(user))
+	fish.set_loc(get_turf(user))
```

Comment:
This seems like an unintentional change?

Reply:
Yeah this looks like it reverts back to the previous behaviour before I made fish attempt to get put in the player's open hand.